### DISCUSSION TITLE
missing package.json file

### DISCUSSION THREAD

**Discussion by jackfusion** [skills/secure-code-game#97] [General]
I am running this on windows 11 and the package.json file is missing from the season foleders.  How can I get a copy of these files?

**Comment by jkcso** [comment]
Hey Jack,

Thanks for letting us know. At the moment, we don't have capacity to fix this but you can play inside a Codespace in under 2 minutes from your browser, where you have 60 hours of free Codespace usage every month. This way it doesn't matter if you have a Windows laptop or another brand. Please follow the instructions on the game to open it in Codespace.
